src/include/OpenImageIO/fmath.h

```diff
@@ -1898,15 +1867,17 @@ T invert (Func &func, T y, T xmin=0.0, T xmax=1.0,
 inline OIIO_HOSTDEVICE float
 interpolate_linear (float x, array_view_strided<const float> y)
 {
+#ifndef __CUDACC__
```

Comment:
Do you think this is something we should address in dassert.h, ensure that the definition of DASSERT is a no-op for Cuda, rather than needing a `#ifndef __CUDACC__` every time it's used? (I realize it's used just once today, but I'm thinking ahead.)

Reply:
That would certainly be cleaner. I'll make the change when I hit the second instance. :smile:

Reply:
Fair enough